arkouda/strings.py

```diff
@@ -525,6 +525,50 @@ def is_title(self) -> pdarray:
             generic_msg(cmd="checkChars", args=f"isTitle {self.objtype} {self.entry.name}")
         )
 
+    @typechecked
+    def strip(self, pattern: Union[bytes, str_scalars] = " ") -> Strings:
+        """
+        Returns a new Strings with all leading and trailing spaces (or set of given characters)
+        stripped from the original.
+
+        Returns
+        -------
+        Strings
+            Strings with all leading and trailing spaces (or set of given characters) stripped from
```

Comment:
This should be updated to something like `Strings  object with all leading and trailing values matching pattern removed."

Reply:
I agree this could use some rewording. I think in cases like this where we are mimicing an existing function, it's often useful to look to their docstring for inspiration. [str.strip](https://docs.python.org/3/library/stdtypes.html#str.strip) says:
>Return a copy of the string with the leading and trailing characters removed. The chars argument is a string specifying the set of characters to be removed. If omitted or None, the chars argument defaults to removing whitespace. The chars argument is not a prefix or suffix; rather, all combinations of its values are stripped:

Reply:
Revised to align with your suggestions.